API/DTOs/ProductDTO.cs

```diff
@@ -1,7 +1,17 @@
-﻿namespace API.DTOs
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace API.DTOs
 {
     public class ProductDTO
     {
         public Guid? Id { get; set; }
+        [Required]
+        public string? Name { get; set; }
+        [Required]
+        public string? Description { get; set; }
+        [Required]
+        public decimal? Price { get; set; }
+        [Required]
+        public int AmountInStock { get; set; }
```

Comment:
geriau daryt nullable propercius ir uzdet required ten kur reikia, nes jei turi ne nullable, tai .NETas pats uzdes defaultines reiksmes jei neatsiusi i requesta, o jei yra nullable su required, tai mes errora. Be to, gal reiktu kazkaip pridet kad amount yra neribotas

Reply:
Ką turi omeny neribotas amount?

Reply:
nu gal nevisada nori uzdet kazkoki amounta, tai gal reiktu padaryt ne required, tada bus nullable ir jei null, reiskia, kad produktas neturi limito